ledger/sandbox/src/main/scala/com/digitalasset/platform/indexer/StandaloneIndexerServer.scala

```diff
@@ -43,7 +42,9 @@ class StandaloneIndexerServer(
         case IndexerStartupMode.ValidateAndStart =>
           startIndexer(indexer, indexerFactory.validateSchema(config.jdbcUrl), actorSystem)
       }
-      _ = loggerFactory.getLogger(getClass).debug("Waiting for indexer to initialize the database")
+      _ = ContextualizedLogger
+        .get[StandaloneIndexerServer]
```

Comment:
I think it'd be nice to pull this out into a field.

Reply:
You're probably right, I assume I was simply sticking with the way the original code worked. I'll work on this, thanks.

Reply:
Addressed by https://github.com/digital-asset/daml/pull/4097/commits/55c498bae0abcbf7ad4e81f858347d369aa1d3c6